library/core/src/pin.rs

```diff
@@ -129,7 +131,7 @@
 //!
 //! To make this work, every element has pointers to its predecessor and successor in
 //! the list. Elements can only be added when they are pinned, because moving the elements
-//! around would invalidate the pointers. Moreover, the [`Drop`] implementation of a linked
+//! around would invalidate the pointers. Moreover, the [`Drop`][Drop] implementation of a linked
```

Comment:
~~Why add this? [`Drop`] should resolve fine on its own.~~ Never mind, this must be https://github.com/rust-lang/rust/issues/80882